.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest 
+
+    steps:
+      - uses: actions/checkout@v2
+      
+    #to speed up the execution time of Github Actions
+      - name: Cache multiple paths  
+        uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: |-
+            vendor
+            node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+                ${{ runner.os }}-yarn-
+     
+      - uses: getong/mariadb-action@v1.1  
+        with:
+          host port: 3306 
+          mariadb version: '10.5' 
+          mysql database: 'test' 
+      
+      - uses: ruby/setup-ruby@v1    
+        with:
+           ruby-version: 2.7.1
+           bundler-cache: true
+
+           
+      - name: Setup Node
+        uses: actions/setup-node@v1
+               
+      - run: 'export PATH=$PATH:`yarn global bin`'
+      
+      - run: export DB=test
+      
+      - run: export SENDER_EMAIL_ADDRESS='sender@wikiedu.org'
+      
+      - name: Installing Packages 
+        run: |
+            yarn install
+            gem install bundler
+            gem install rubocop
+            gem install rubocop-rspec-focused
+            bundle config set path 'vendor'
+            bundle install --jobs=3 --retry=3
+            
+      - run: mkdir tmp -m 777
+      - run: cp config/database.example.yml config/database.yml
```

Comment:
Rather than changing the username/password in the example file to match what GitHub Actions expects, I think we should add a new file explicitly for this Actions workflow and use that instead of the example file.

Reply:
so like making a new file with the same content except username and password and then naming it something else?